bumble/device.py

```diff
@@ -2743,20 +2743,6 @@ def on_connection_authentication_failure(self, connection, error):
         )
         connection.emit('connection_authentication_failure', error)
 
-    @host_event_handler
-    @with_connection_from_address
-    def on_ssp_complete(self, connection):
-        # On Secure Simple Pairing complete, in case:
-        # - Connection isn't already authenticated
-        # - AND we are not the initiator of the authentication
-        # We must trigger authentication to know if we are truly authenticated
-        if not connection.authenticating and not connection.authenticated:
```

Comment:
Even if this was dead code I think it still remains valid, what do you think?

Reply:
I highly disencourage this behavior, because it may raise authentication collision, and it's not something listed on the spec